examples/megacli.cpp

```diff
@@ -3907,6 +3942,26 @@ void exec_sync(autocomplete::ACState& s)
         }
     }
 }
+
+void exec_syncconfig(autocomplete::ACState& s)
+{
+    if (s.words.size() == 1)
+    {
+        cout << "Current sync config: " << syncConfig.syncType <<
+                " " << syncConfig.syncDeletions << " " << syncConfig.forceOverwrite << endl;
+    }
+    else if (s.words.size() == 4)
+    {
+        syncConfig.syncType = static_cast<SyncConfig::Type>(atoi(s.words[1].s.c_str()));
+        syncConfig.syncDeletions = atoi(s.words[2].s.c_str());
+        syncConfig.forceOverwrite = atoi(s.words[3].s.c_str());
+        cout << "Successfully applied new sync config!" << endl;
```

Comment:
I'd say that a sanity check here would make sense: if the sync type is full, the user doesn't need to add the flags, since they will not be used at all. In other words, the flags have to be provided only for `up` and `down` types, and for a `full` sync the command could report an error message to avoid setting an invalid configuration.